src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -162,7 +162,6 @@ public String getJsonString() {
     
     @Override
     public void sanitizeForSaving() {
-        this.feedbackSessionName = Sanitizer.sanitizeTitle(feedbackSessionName);
```

Comment:
@bamboo3250, how should we handle sanitization (both for security and for cleaning) consistently across the code?